routes.js

```diff
@@ -0,0 +1,47 @@
+const express = require("express");
+const app = express();
+const port = 8080;
+
+const data = [
+  {
+    id: 1,
+    item: "Toilet Paper",
+    person: "Kesha",
+    country: "USA",
+    image_url:
+      "https://images.news18.com/ibnlive/uploads/2023/03/whatsapp-image-2023-03-04-at-3.12.13-pm.jpeg?impolicy=website&width=510&height=356",
+    description:
+      "Kesha revealed on a talk show that she had a strange habit of eating toilet paper.",
+  },
+];
+
+app.use(express.json());
+
+app.get("/", (req, res) => {
+  res.send(data);
+});
+
+app.post("/", (req, res) => {
```

Comment:
Consider adding validation for the incoming request bodies in the POST and PUT routes. This can help to ensure that the data being sent to your server is in the correct format and contains all the necessary information. You can use libraries like 'joi' for this purpose. [important]